pexpect/__init__.py

```diff
@@ -68,7 +68,12 @@
 
 from .exceptions import ExceptionPexpect, EOF, TIMEOUT
 from .utils import split_command_line, which, is_executable_file
-from .pty_spawn import spawn, spawnu, PY3
+
+if sys.platform == 'win32':
+    from .popen_spawn import (
+        PopenSpawn as spawn, PopenSpawnUnicode as spawnu, PY3)
+else:
+    from .pty_spawn import spawn, spawnu, PY3
```

Comment:
Sorry: I don't think it's a good idea for `pexpect.spawn` to be a different class depending on which platform it's running on. I'd rather `pexpect.spawn` simply wasn't available on Windows.

If a downstream user knows that, for their particular use case, it makes sense to use a pty-based spawn on Unix and a Popen-based spawn on Windows, they can have conditional imports in their own code. But I don't want to make that automatic.